test_d3m_config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "dataset_schema": "/data/d3m/o_185/data/dataSchema.json",
+    "problem_schema": "/data/d3m/o_185/problemSchema.json",
+    "training_data_root": "/data/d3m/o_185/data",
+    "pipeline_logs_root": "/data/d3m-logs",
+    "executables_root": "/data/d3m-tmp",
+    "temp_storage_root": "/data/d3m-tmp",
+    "test_data_root": "/data/d3m/o_185/data",
+    "results_path": "/data/d3m-tmp/model_result.csv"
```

Comment:
I think results_path should point to a folder, shouldn't it? The output depends on the chosen pipeline and I don't think we can rely on a single output CSV for all the pipelines. IIRC each pipeline would have its own result file.

Reply:
Results_path is the result of the test executable, so I think it should be a single file. It's not very clear in the specification, what do you think?

https://datadrivendiscovery.org/wiki/display/gov/TA2+Configuration+file+syntax

Reply:
The thing is each pipeline produces one executable, and we may have multiple pipelines and thus get multiple executables. When TA3 talks to TA2, we actually send {executables_root}/{pipeline_id}, so it doesn't really matter. Semantically I'd prefer taking results_path as a directory.

Reply:
Oh on second thought I see that I may have missed something. We do have {exectuables_root} in TA2 test, and results_path points to the answer file (or say the Test Executable output). Then I guess it's fine here to have a CSV. I'm also assuming, right now we only require one pipeline's answer file. If we have multiple answer files then this may need to be directory again.